app/components/layouts/SecondaryNavigation.module.scss

```diff
@@ -0,0 +1,7 @@
+@import "app/styles/utils/_helpers.scss";
+
+.container {
+  background: $surface-4;
+  padding: $spacing-6 $spacing-16;
```

Comment:
On smaller breakpoints, the button should be aligned to the left/start with a smaller padding `$spacing-4`. https://www.figma.com/design/iDBeNK7SnZXF8wc3pbPTCY/Our415-Site?node-id=2659-23607&m=dev

Reply:
Nice spot. The fact there's not a page-level container setting width/padding for every block is challenging!